### DISCUSSION TITLE
Can we get rid of webpack?

### DISCUSSION THREAD

**Discussion by ottopaulsen** [vuepress/core#1430] [Q&A] [answered]
We are trying to remove webpack totally from out project, but we still get it included, seemingly because we are using theme-default. Is it possible to get totally rid of webpack (still using theme-default)?

**Comment by meteorlxy** [comment, marked as answer]
You won't install any webpack related packages by default, unless you install `vuepress-webpack` explicitly. Why do you ask this question?

**Reply by ottopaulsen** [reply]
We are trying to remove all that has with webpack, for example `sass-loader`, and when we run `npm list sass-loader` we can see that  `@vuepress/theme-default@2.0.0-beta.67` depends on it.

**Reply by Mister-Hope** [reply]
That's an **optional** dependency, you won't install it with modern package manager and correct option.

E.g.: https://github.com/miniapp-tool/mptool/blob/main/pnpm-lock.yaml

![image](https://github.com/vuepress/vuepress-next/assets/33315834/38b924a4-dbfd-4f60-83cb-86639ec4784f)

**Reply by ottopaulsen** [reply]
Ok. Thanks!
